packages/native/src/components/range/RangeSlider.js

```diff
@@ -172,8 +172,10 @@ class RangeSlider extends Component {
 			this.setState({
 				currentValue,
 			}, () => {
-				this.updateQuery([currentValue[0], currentValue[1]], props);
+				const rangeValues = [currentValue[0], currentValue[1]];
+				this.updateQuery(rangeValues, props);
 				this.locked = false;
+				if (props.onValueChange) props.onValueChange(rangeValues);
```

Comment:
the format for the onValueChange's param here is wrong:

```
{
	start: currentValue[0],
	end: currentValue[1],
}
```

For determining the pattern, please check the second param in `checkValueChange`. Also, it would be great if you can fix this for web as well.